src/components/SideBar/SideBarComponents/FacetFilters.js

```diff
@@ -96,7 +96,7 @@ export const FacetPanelComponent = ({ classes }, ref) => {
       toggleSlider(value, sideBarItem);
     }
   };
-  // data from store for sidebar laoding
+  // daa from store for sidebar laoding
```

Comment:
Please check here.

Reply:
dont remember making that change, might have hit backspace on error. Should be fixed now